src/liballoc/tests/btree/map.rs

```diff
@@ -783,11 +796,11 @@ fn test_clone() {
         assert_eq!(map, map.clone());
     }
 
-    // Full 2-level and minimal 3-level tree (sizes 143, 144 -- the only ones we clone for).
-    for i in 1..=144 {
+    // Also test two big trees.
+    for i in 1..=MIN_INSERTS_HEIGHT_2 {
         assert_eq!(map.insert(i, i), None);
         assert_eq!(map.len(), i);
-        if i >= 143 {
+        if i > MIN_INSERTS_HEIGHT_2 - 2 {
```

Comment:
I have to say I found `i >= MIN_INSERTS_HEIGHT_2-1` clearer.

Reply:
I don't like either much and wonder why I worshipped that loop for so long. How about this? Also no more gibberish about the demised `NodeHeader`.